js/main.js

```diff
@@ -29,5 +32,97 @@ function submitForm(event) {
   data.entries.unshift(inputValues);
   data.nextEntryId++;
   newEntryForm.reset();
+
+  var newEntry = renderEntry(data.entries[0]);
+  ulElement.prepend(newEntry);
+
+  entriesView.className = 'view';
+  entryFormView.className = 'view hidden';
+  data.view = 'entries';
```

Comment:
Once you revise `switchView` in the way I described, you'll be able to just call that function here.